cmd/vcluster/cmd/certs/rotate.go

```diff
@@ -0,0 +1,69 @@
+package certs
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/loft-sh/log"
+	"github.com/loft-sh/vcluster/pkg/certs"
+	"github.com/loft-sh/vcluster/pkg/config"
+	"github.com/loft-sh/vcluster/pkg/constants"
+	"github.com/loft-sh/vcluster/pkg/pro"
+	"github.com/loft-sh/vcluster/pkg/setup"
+	"github.com/spf13/cobra"
+)
+
+type rotateCmd struct {
+	log log.Logger
+}
+
+func rotate() *cobra.Command {
+	cmd := &rotateCmd{
+		log: log.GetInstance(),
+	}
+
+	rotateCmd := &cobra.Command{
+		Use:   "rotate",
+		Short: "Rotates control-plane client and server certs",
```

Comment:
Do you think it would be worthwhile to add a "Long" description to these as well, like [this one](https://github.com/loft-sh/vcluster/blob/main/cmd/vclusterctl/cmd/platform/start.go#L43-L57)

Reply:
Ah, just scrolled down and saw that you have the "long" on a few others.